src/chart/main/chart.cpp

```diff
@@ -105,28 +105,24 @@ void Chart::draw(Gfx::ICanvas &canvas) const
 	if (actPlot
 	    && (!events.draw.begin
 	        || events.draw.begin->invoke(
-	            Util::EventDispatcher::Params{}))) {
+	            Util::EventDispatcher::Params{}))) 
+	{
+		Draw::DrawingContext context(canvas, layout, events.draw, *actPlot);
+
 		Draw::drawBackground(
```

Comment:
Isn't the drawBackground and the drawLabel a DrawingContext too?

Reply:
Yes, they will be changed in one of the a following PRs